contractsV5/mock/MockDAO.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity 0.5.11;
+
+import "../IKyberDAO.sol";
+import "../UtilsV5.sol";
+
+contract MockDAO is IKyberDAO, Utils {
+
+    uint public rewardInBPS;
+    uint public rebateInBPS;
+    uint public epoch;
+    uint public expiryBlockNumber;
+
+    constructor(uint _rewardInBPS, uint _rebateInBPS, uint _epoch, uint _expiryBlockNumber) public {
+        rewardInBPS = _rewardInBPS;
+        rebateInBPS = _rebateInBPS;
+        epoch = _epoch;
+        expiryBlockNumber = _expiryBlockNumber;
+    }
+
+    function setMockValues(uint _rewardInBPS, uint _rebateInBPS, uint _epoch, uint _expiryBlockNumber) public {
+        rewardInBPS = _rewardInBPS;
+        rebateInBPS = _rebateInBPS;
+        epoch = _epoch;
```

Comment:
better have separate API for setting epoch and expiry block.
notice epoch also used for network fee....

Reply:
done